<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_Controller</class>
 <widget class="QMainWindow" name="C_Controller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>C_Controller</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	border-image:url(:/images/background.png);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Germania</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget{
	background-color:rgb(250,0,0);
}

QTabWidget::pane {
    background: transparent;
    border:none;
}

QTabBar::tab {
    background: transparent;
	border:none;
}



QWidget{background:transparent} QTabWidget::pane{border: none;background-color: transparent;} 
QTabBar::tab {background-color: #1E1E1E;color: white;width: 233px;height:40px;font-size:24px} 
 QTabBar::tab:hover{background-color:#303030; color: white;}    
QTabBar::tab:selected{background-color: #151515; color: #f3c216;}
</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Commandes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1,1,4">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>150</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>150</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonStartGame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Harry Potter</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color:#EE151515;
	color:white;
	border-radius : 16px;
}

QPushButton:hover{
	background-color:#F8151515;
}</string>
          </property>
          <property name="text">
           <string>Demarrer la partie</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRestartTurn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Harry Potter</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color:#EE151515;
	color:white;
	border-radius : 16px;
}

QPushButton:hover{
	background-color:#F8151515;
}</string>
          </property>
          <property name="text">
           <string>Recommencer le tour</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="styleSheet">
        <string notr="true">QWidget{
	background-color:rgba(0,0,0,0);
	border: none;
}
</string>
       </property>
       <attribute name="title">
        <string>Suivi de la partie</string>
       </attribute>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>350</y>
          <width>701</width>
          <height>511</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaPlayers">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>701</width>
           <height>511</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QAbstractScrollArea
{
background-color: transparent;
}
QWidget#scrollAreaWidgetContents{
background-color: transparent; /*or a colour*/
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="layoutPlayers">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="GUI_LawBoard" name="Board_DeathEater" native="true">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>180</y>
          <width>450</width>
          <height>160</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>160</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>160</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image:url(:/images/board_deathEater_56.png);</string>
        </property>
       </widget>
       <widget class="GUI_Stack" name="stack" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>200</width>
          <height>300</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="GUI_LawBoard" name="Board_PhenixOrder" native="true">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>450</width>
          <height>160</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>160</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>160</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image:url(:/images/board_phenixOrder.png);</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <property name="styleSheet">
        <string notr="true">QWidget{
	background-color:rgba(0,0,0,0);
	border: none;
}
</string>
       </property>
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="output_log">
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextBrowser{
	background-color:rgba(250,250,250,200);
	color:#111111;
	border:none;
}</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Nirmala UI'; font-size:10pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Sans Serif'; font-size:15pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUI_Stack</class>
   <extends>QWidget</extends>
   <header>GUI/servergui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GUI_LawBoard</class>
   <extends>QWidget</extends>
   <header>GUI/servergui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
